code/datums/status_effects/debuffs.dm

```diff
@@ -968,3 +968,53 @@
 /datum/status_effect/cloudstruck/Destroy()
 	. = ..()
 	QDEL_NULL(mob_overlay)
+
+/datum/status_effect/pranked
+	id = "pranked"
+	duration = 20 SECONDS
+	status_type = STATUS_EFFECT_REFRESH
+	alert_type = null
+	///increments every time the status effect is refreshed. Higher counter == more powerful clown combos.
+	var/prank_counter = 1
+
+/datum/status_effect/pranked/refresh()
+	. = ..()
+	prank_counter++
+/datum/status_effect/slapped_silly
+	id = "smacked_silly"
+	duration = 10 SECONDS
+	///This sound is played for the duration of the effect.
+	var/datum/looping_sound/dizzy_birdies/bird_noise
+	///This overlay is applied to the owner for the duration of the effect.
+	var/mutable_appearance/mob_overlay
```

Comment:
convert this to a static appearance so it is only created once

https://github.com/tgstation/tgstation/blob/master/.github/CONTRIBUTING.md#icons-are-for-image-manipulation-and-defining-an-objs-icon-var-appearances-are-for-everything-else